bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -810,7 +810,7 @@ Set net or gross asslimiation for the FATES stomatal model
 </entry>
 
 <entry id="fates_stomatal_model" type="char*256" category="physics"
-        group="clm_inparm" valid_values="ballberry, medlyn" value="ballberry">
+        group="clm_inparm" valid_values="ballberry1987, medlyn2011" value="ballberry1987">
```

Comment:
@ckoven @rgknox is `1987` the correct year for `ballberry`?  I couldn't find the specific data cross referencing the fates tech doc.